x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -386,15 +386,15 @@ export const HostList = () => {
               <EuiTitle size="l">
                 <h1 data-test-subj="pageViewHeaderLeftTitle">
                   <FormattedMessage
-                    id="xpack.securitySolution.hostList.pageTitle"
-                    defaultMessage="Hosts"
+                    id="xpack.securitySolution.endpointList.pageTitle"
+                    defaultMessage="Endpoints"
                   />
                 </h1>
               </EuiTitle>
             </EuiFlexItem>
             <EuiFlexItem grow={false}>
               <EuiBetaBadge
-                label={i18n.translate('xpack.securitySolution.endpoint.hostList.beta', {
+                label={i18n.translate('xpack.securitySolution.endpoint.endpointList.beta', {
```

Comment:
Unrelated to you change: maybe remove `.endpoint.` from this id. Maybe it was left over. I think all ids in this module should have the same prefix.

Reply:
Happy to remove it here. I don't know what's considered a module (file? folder? parent folder? subfolders?) but I will say these i18n keys vary a bit. I searched for "xpack.securitySolution.endpoint" and came up with.. lots.

There's also a lot of inconsistencies overall. See:

`i18n.translate('xpack.securitySolution.endpoint.hostDetails.policyResponse.load_config', {`

and

`title: i18n.translate('xpack.securitySolution.endpoint.host.details.policyStatus', {`


note `hostDetails` and `host.details`. Perhaps we should make a backlog ticket for auditing these and use consistent namespaces?

Reply:
I'm ok with opening another issue to track. This is left over from when we moved from the `endpoint` plugin to SIEM (not Security_Solution) 🙂 
I think we're free to come up with our own conventions if we see it necessary. At a minimum, `xpack.securitySolution` should be on all entries - I think this makes it clear (when these are all extracted from Kibana) which plugin owns what. After that value, we can come up with anything we want.  My main point in my prior comment was that within a single module (file), all the keys should at least have the same prefix.